### PR TITLE
Parse RPT series without document sequence numbers

### LINKED ISSUE #101: Parsing NBS RPT series without document numbers
More details here [relaton-nist#59](https://github.com/relaton/relaton-nist/issues/59)
#94

NBS RPT ADHOC
NBS report ; Apr-Jun1948
NBS report ; Apr-Jun1949
NBS report ; Apr-Jun1950
NBS report ; Apr-Jun1951
NBS report ; Jan-Jun1971
NBS report ; Jan-Mar1948
NBS report ; Jan-Mar1950
NBS report ; Jul-Dec1947
NBS report ; Jul-Sep1949
NBS report ; Jul-Sep1950
NBS report ; Oct-Dec1949
NBS report ; Oct-Dec1950
NBS report ; div9

@ronaldtse What do we do with them? Should we use publication date as number?

### PR DESCRIPTION
**Pull request by mico**
closes #101

### CODE REVIEW

File: spec/nist_pubid/document_spec.rb [hunk 1]
```diff
@@ -917,6 +918,27 @@
...
+    context "NBS RPT ADHOC" do
+      let(:original_pubid) { "NBS RPT ADHOC" }
+      let(:short_pubid) { "NBS RPT ADHOC" }
+
+      it_behaves_like "converts pubid to different formats"
+    end
+
+    context "NBS report ; div9" do
+      let(:original_pubid) { "NBS report ; div9" }
+      let(:short_pubid) { "NBS RPT div9" }
```
Comment (ronaldtse) [inline spec/nist_pubid/document_spec.rb:937]: I think we need more research into these numbers. "NBS report ; div9" is actually this document:
https://www.govinfo.gov/app/details/GOVPUB-C13-43251ad869fb41acd5a49a05afe80772

Which is actually "NBS RPT 3119".
Screenshot 2022-01-25 at 7 51 20 AM

For all these strange entries we should just Google them to find the correct document. Seems that they are mostly online.

**Review by ronaldtse — CHANGES_REQUESTED** [review]

File: spec/nist_pubid/document_spec.rb (hunk 1 above)
Comment (mico) [reply spec/nist_pubid/document_spec.rb:937]: > Screenshot 2022-01-25 at 7 51 20 AM

@ronaldtse I cannot find this document, by following your link I see another document:
image

Tried to google another documents, didn't find any numbers.

**Comment by ronaldtse** [comment]
@mico ah you're right. Don't know why I got sent to another document. In this case it is a report without a report number...

File: spec/nist_pubid/document_spec.rb (hunk 1 above)
Comment (ronaldtse) [reply spec/nist_pubid/document_spec.rb:937]: Agree with just using div9...

**Comment by ronaldtse** [comment]
@mico please help merge. Thanks!

**Pull request merged**